src/components/SignUp/Landing.tsx

```diff
@@ -252,4 +318,27 @@ const LoginLink = styled.p`
   }
 `;
 
+const CardHeading = styled.h3`
+  width: 225px;
+  top: 24%;
+  bottom: 43%;
+
+  font-family: Montserrat;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 28px;
+  /* identical to box height, or 117% */
+  letter-spacing: 0.07em;
+  flex: none;
```

Comment:
I think flex, order, and flex-grow are all unnecessary here as well

Reply:
Yeah, the flex options and top/bottom were all extraneous, so they've been removed.